.github/dependabot.yml

```diff
@@ -182,7 +182,7 @@ updates:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
-    directory: "addons/packages/vsphere-cpi/1.22.4/test"
+    directory: "addons/packages/vsphere-cpi/1.22.5/test"
```

Comment:
I think we actually want to keep this line as is and add _another_ chunk that corrisponds to the new package being added. We still want to have the existing chunk for v1.22.4 _in addition_ to the one for v1.22.5

Reply:
Oh, what's this file used for?
Let me update it

Reply:
It's used to configure GitHub's dependabot, an automated dependency scanning / updating system. These configs ignore tanzu-framework which the TCE team has been bumping manually